.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +34,20 @@
 
 ## RSS or Atom Feed 
 
+* [ ] The content with feed.type = dotnet must be .NET related:
+* .NET itself
+* .NET libraries
+* .NET languages C#, F#, Visual Basic
+* Tools needed during development (for .NET)
+* CI/CD
+* SPA Frontend technologies like Angular, Vue.js, React etc.
```

Comment:
add: "in combinaton with .NET" we dont want pure angular or react stuff if theres not at least some relation to .net

Reply:
Resolved.